src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/RuleBuilder.cs

```diff
@@ -132,16 +132,18 @@ public void AddRule(string rule)
                             condition.Match = new IsFileMatch(input.Invert);
                             break;
                         case OperationType.SymbolicLink:
-                            throw new NotImplementedException("Symbolic links are not implemented");
+                            throw new NotSupportedException("Symbolic links are not supported because " +
```

Comment:
NIE. Add a todo to see if [`FileAttributes.ReparsePoint`](https://docs.microsoft.com/en-us/dotnet/core/api/system.io.fileattributes#System_IO_FileAttributes_ReparsePoint) will work. We'll need to verify if its semantically correct for junctions, hard links, sym links, etc